src/celastro/date.cpp

```diff
@@ -213,13 +210,14 @@ Date::toString(Format format) const
     switch(format)
     {
     case Locale:
-        return fmt::format(std::locale(), "{:%c}"sv, cal_time);
+        return fmt::format(loc, "{:%c}"sv, cal_time);
     case TZName:
-        return fmt::format(std::locale(), "{:%Y %b %d %H:%M:%S %Z}"sv, cal_time);
+        return fmt::format(loc, "{:%Y %b %d %H:%M:%S %Z}"sv, cal_time);
     default:
-        return fmt::format(std::locale(), "{:%Y %b %d %H:%M:%S %z}"sv, cal_time);
+        return fmt::format(loc, "{:%Y %b %d %H:%M:%S %z}"sv, cal_time);
     }
 #else
+    static auto monthAbbreviations = MonthAbbreviations(loc);
```

Comment:
this allocates heap memory, so it should not be used this way. add a static pointer and use `new` on the 1st call